music_assistant/server/providers/jellyfin/__init__.py

```diff
@@ -153,41 +151,26 @@ async def get_config_entries(
 class JellyfinProvider(MusicProvider):
     """Provider for a jellyfin music library."""
 
-    # _jellyfin_server : JellyfinClient = None
-
     async def handle_async_init(self) -> None:
         """Initialize provider(instance) with given configuration."""
+        session_config = SessionConfiguration(
+            session=self.mass.http_session,
```

Comment:
I was thinking if we also need a separate aiohttp session at core level without ssl, just like HA which also has 2.
And then some provider config setting "do not verify ssl" which then uses that ssl-less http session

Reply:
I've been lucky and not needed the unverified client session over there. Can look into what HA does and then do similar in a follow up?

Reply:
Yeah it was just an idea - there was also a request somewhere from somebody who asked if we can offer an "do not verify ssl" option on the home Assistant provider in MA as he is using a self signed cert.
Definitely for later (just thinking out loud) - now we just have a singe aiohttp session in the server